test/core/public/configure.js

```diff
@@ -239,4 +239,230 @@ describe('axe.configure', function() {
 		assert.equal(axe._audit.rules[3].id, 'black-panther');
 		assert.equal(axe._audit.rules[3].enabled, true);
 	});
+
+	describe('given a locale object', function() {
```

Comment:
I think we need tests for:

1. An axe.configure call with both metadata and a locale, make sure that locale is always used over metadata (should it throw if we're trying to set both?)
2. Test that if you pass an object into the message function, the template ran correctly.

Reply:
Tests have been added.